apps/web/app/[language]/homestead/layout.tsx

```diff
@@ -12,6 +12,7 @@ export default function HomesteadLayout({ children }: { children: ReactNode }) {
         { segment: 'cats', label: <Trans id="homestead.cats"/> },
         { segment: 'decorations', label: <Trans id="homestead.decorations"/> },
         { segment: 'glyphs', label: <Trans id="homestead.glyphs"/> },
+        { segment: 'refinedMaterials', label: <Trans id="homestead.refinedMaterials"/> },
```

Comment:
Lets add the new page between decorations and glyphs, because the materials somehow belong to decorations?

I also prefer just `/homestead/materials` as the URL. We don't have any other materials related to the homestead, so lets keep it short and clean.

Reply:
good idea